### PR TITLE
Feature/update table

### PR DESCRIPTION
**Pull request by hamidazim321**
# Queries and Results
**Query: Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that the species columns went back to the state before the transaction.**
**Result:**
q1 1
q1 2

**Query:Inside a transaction:
- Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
- Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
- Verify that changes were made.
- Commit the transaction.
- Verify that changes persist after commit.**
**Results:**
q2 1
q2 3

**Query:
- Inside a transaction delete all records in the animals table, then roll back the transaction.
- After the rollback verify if all records in the animals table still exists**
**Results:**
q3 1
q3 2

**Query: Inside a transaction:
- Delete all animals born after Jan 1st, 2022.
- Create a savepoint for the transaction.
- Update all animals' weight to be their weight multiplied by -1.
- Rollback to the savepoint
- Update all animals' weights that are negative to be their weight multiplied by -1.
- Commit transaction**
**Results:**
q4 1
q4 2
q4 3
q4 4

**Query: How many animals are there?**
**Result:**
q5

**Query: How many animals have never tried to escape?**
**Result:**
q6

**Query: What is the average weight of animals?**
**Result:**
q7

**Query: Who escapes the most, neutered or not neutered animals?**
**Result:**
q8

**Query: What is the minimum and maximum weight of each type of animal?**
**Result:**
q9

**Query: What is the average number of escape attempts per animal type of those born between 1990 and 2000?**
**Result:**
q10

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,6 +1,15 @@
...
-('Agumon', '2020-02-03', 10.23, TRUE, 0),
+(' Charmander', '2022-02-08', 10.23, TRUE, 0),
 ('Gabumon', '2018-11-15', 8.0, TRUE, 2),
 ('Pikachu', '2021-01-07', 15.04, FALSE, 1),
-('Devimon', '2017-05-12', 11.0, TRUE, 5);
\ No newline at end of file
+('Devimon', '2017-05-12', 11.0, TRUE, 5);
+
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) VALUES
+('Charmander', '2020-02-03', -11.0, FALSE, 0),
```
Comment (BregornOriginal) [inline data.sql:9]: - [ ] Kindly fix the `date_of_birth` of Charmander to match with the one provided.

File: data.sql [hunk 2]
```diff
@@ -1,6 +1,15 @@
...
+('Plantmon', '2021-11-15', -5.7, TRUE, 2),
+('Squirtle', '1993-04-02', -12.13, FALSE, 3),
+('Angemon', '2005-06-12', -5.0, TRUE, 1),
```
Comment (BregornOriginal) [inline data.sql:12]: - [ ] Kindly fix the `weight_kg` of Angemon to match with the one provided.

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
Hi @hamidazim321  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

giphy (4)

## Highlights

✅ Linters are passing
✅ Excellent commit history
✅ Good work implementing queries 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TuyishimireEric — APPROVED** [review]
# Status: APPROVED 🟢  
### Hi  @hamidazim321  👋 👋 

#### I'm @TuyishimireEric, Your code reviewer,
Good job on implementing the project requirements! 🥇 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it 🦾  
Congratulations! 🎉

## Highlight 👍 
✅  Made suggested changes

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/enus/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**